flatpak/manifest_tool/tests/flutter/test_offline_fixes.py

```
@@ -1,15 +1,15 @@
+import pytest  # noqa: F401
 from pathlib import Path
 import sys
-import tempfile
+import tempfile  # noqa: F401
 
 # Add parent directory to path for imports
 sys.path.insert(0, str(Path(__file__).parent.parent.parent))
 
-from manifest_tool.core.manifest import ManifestDocument
-from manifest_tool.flutter import offline_fixes
+from manifest_tool.core.manifest import ManifestDocument  # noqa: E402
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)